js/components/graph/InfoBox.js

```diff
@@ -5,10 +5,7 @@ export default class InfoBox extends React.Component {
 
   render() {
     if (this.props.showInfoBox) {
-      //TODO: move to CSS
       var gStyles = {
-        cursor: "pointer",
-        transition: "opacity .4s",
         opacity: this.props.showInfoBox ? 1 : 0
```

Comment:
Taking a second look at this code, the opacity is functionally hardcoded to 1, as the opacity is only set when `this.props.showInfoBox` is true, meaning it's only ever set to 1. The way we are making the infobox disappear is by returning an empty HTML <g> tag if `this.props.showInfoBox` is false. 

Should I refactor it so it's visibility is determined by css, so that I can try out that CSS transitions based approach you mentioned?

Reply:
Sorry I missed this question, but yes the visibility should be determined only by CSS, and not by this `opacity` rule here.

Reply:
Ok. I've made the change, but if you hover over a course till the Infobox fades in, and then you hover over another, the Infobox doesn't fade in, it just appears. 

I think this isn't ideal. Do you want me to take a stab at it in this PR, or wait till a separate one?